### PR TITLE
add ogrn

### PR DESCRIPTION
**Pull request by arnstadm**
## Description

Added OGRN pcb to QMK

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/akb/ogrn/keymaps/via/ogrn.json [hunk 1]
```diff
@@ -0,0 +1,332 @@
+{
```
Comment (drashna) [inline keyboards/akb/ogrn/keymaps/via/ogrn.json:1]: This appears to be a VIA json. 
https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs

> Do not include VIA json files in the PR. These do not belong in the QMK repository as they are not used by QMK firmware – they belong in the [VIA Keyboard Repo](https://github.com/the-via/keyboards)

Also, you may have noticed that "via.json" files are not included by default, and you probably had to rename the file to get it to be added.  This is intentional configuration design, as VIA json files do not belong in the QMK repo.

File: keyboards/akb/ogrn/readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# akb/ogrn
+
+![akb/ogrn](https://alchemistkeyboards.com/wp-content/uploads/2022/05/OGR_N_Dolch_CROP-scaled.jpg)
```
Comment (drashna) [inline keyboards/akb/ogrn/readme.md:3]: Please rehost on Imgur.

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by keyboard-magpie — APPROVED** [review]

File: keyboards/akb/ogrn/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,45 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/akb/ogrn/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/akb/ogrn/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,45 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/akb/ogrn/keymaps/via/keymap.c:1]: Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/akb/ogrn/readme.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# akb/ogrn
+
+![akb/ogrn](https://i.imgur.com/qbolyMa.png)
```
Comment (drashna) [inline keyboards/akb/ogrn/readme.md:3]: This uses a lower resolution (but still pretty high) image that is more suitable with markdown rendering.
```suggestion
![akb/ogrn](https://i.imgur.com/qbolyMal.png)
```

File: keyboards/akb/ogrn/info.json [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+        "num_lock": "F4"
+    },
+    "processor": "atmega32u4",
+    "url": "https://alchemistkeyboards.com/projects/keyboards/ogr/",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x4F4E",
+        "vid": "0x414B"
+    },
+    "layouts": {
```
Comment (drashna) [inline keyboards/akb/ogrn/info.json:33]:
```suggestion
  "community_layouts": [
    "numpad_5x4"
  ],
    "layouts": {
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**